bedrock/settings/base.py

```diff
@@ -881,26 +881,31 @@ def get_app_name(hostname):
 DONATE_LINK_UNKNOWN = "https://donate.mozilla.org/?utm_source=mozilla.org&utm_medium=referral&utm_content={source}"
 
 DONATE_PARAMS = {
+    "aud": {"currency": "aud", "symbol": "A$", "presets": "60,30,20,10", "default": "30", "prefix": "true"},
     "brl": {"currency": "brl", "symbol": "R$", "presets": "80,40,20,10", "default": "40", "prefix": "true"},
-    "cad": {"currency": "cad", "symbol": "$", "presets": "65,30,15,4", "default": "30", "prefix": "true"},
+    "cad": {"currency": "cad", "symbol": "$", "presets": "60,30,20,10", "default": "30", "prefix": "true"},
+    "chf": {"currency": "chf", "symbol": "Fr.", "presets": "60,30,20,10", "default": "30", "prefix": "true"},
```

Comment:
Suggestion: change the symbol for Swiss Francs to `CHF` to match the donation site (this is also what we use for VPN)
```suggestion
    "chf": {"currency": "chf", "symbol": "CHF", "presets": "60,30,20,10", "default": "30", "prefix": "true"},
```